src/records/RecCore.cc

```diff
@@ -244,7 +244,8 @@ RecCoreInit(Diags *_diags)
     RecLog(DL_Note, "records parsing completed.");
     if (!err.empty()) {
       std::string text;
-      RecLog(DL_Note, "%s", swoc::bwprint(text, "{}", err).c_str());
+      RecLog(DL_Warning, "%s",
+             swoc::bwprint(text, "We have found the following issues when reading the records node:\n {}", err).c_str());
     }
   } else {
     RecLog(DL_Note, "%s does not exist.", g_rec_config_fpath);
```

Comment:
should this be a warning as well? if records.yaml doesn't exist?

Reply:
The [above](https://github.com/apache/trafficserver/pull/11560/files#diff-337da9a9ab2e23a59f3648a7015d48326c40944682b481f12d10260df2cda794L238) warning should be already displayed. 

I think eventually down the line we may want to make all this more strict.. or maybe not.